Cargo.toml

```diff
@@ -51,18 +51,18 @@ derive_more = "0.99.17"
 dialoguer = "0.11"
 digest = "0.10.5"
 directories = "5"
-fil_actor_account_state = "7.0.0-rc.5"
-fil_actor_cron_state = "7.0.0-rc.5"
-fil_actor_datacap_state = "7.0.0-rc.5"
-fil_actor_init_state = "7.0.0-rc.5"
-fil_actor_interface = "7.0.0-rc.5"
-fil_actor_market_state = "7.0.0-rc.5"
-fil_actor_miner_state = "7.0.0-rc.5"
-fil_actor_power_state = "7.0.0-rc.5"
-fil_actor_reward_state = "7.0.0-rc.5"
-fil_actor_system_state = "7.0.0-rc.5"
-fil_actor_verifreg_state = "7.0.0-rc.5"
-fil_actors_shared = "7.0.0-rc.5"
+fil_actor_account_state = "7.0.0"
```

Comment:
This will probably clash with #3739

Reply:
It might, we'll have several PRs depeding on this version bump.